packages/core-components/src/components/Select/Select.tsx

```diff
@@ -101,21 +102,23 @@ const useStyles = makeStyles(
   { name: 'BackstageSelect' },
 );
 
-type Item = {
+export type SelectItem = {
```

Comment:
To get rid of the warnings we also probably want to add the @public annotation. You'll see they're everywhere in most places. :)

Reply:
Added public annotation to types and component 👍  . But - how do you see these warnings? :o I did't see them neither in the terminal nor in IDE 🤔

Reply:
Now i see them - you mean warnings in the docs? :)